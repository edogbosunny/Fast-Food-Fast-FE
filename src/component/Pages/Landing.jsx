/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
import ProductCard from '../UI/cards/ProductCard.jsx';
import ProductTable from '../UI/tables/ProductTable.jsx';
import '../../styles/ind.css';

class Landing extends Component {
  render() {
    return (
      <div>
        <div className='bg-pics' />
        <div className='caption-div '>
          <h1 className='caption mb-4 mt-5'>Order Food Fast!</h1>
          <p className='caption-small lead'>Get your food in minutes!</p>
        </div>
        <div>
          <h1 className='order'>Available Order</h1>
        </div>

        <ProductTable />
        <ProductCard />
      </div>

    );
  }
}

export default Landing;
