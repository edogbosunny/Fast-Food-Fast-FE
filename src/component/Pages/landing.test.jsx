import { shallow } from 'enzyme';
import React from 'react';
import Landing from './Landing';

describe('Login Container', () => {
  const wrapper = shallow(
    <Landing />,
  );

  it('should mount the login page Yeah!', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
