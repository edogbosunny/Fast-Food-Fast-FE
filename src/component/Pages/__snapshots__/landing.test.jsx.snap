// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Container should mount the login page Yeah! 1`] = `
<div>
  <div
    className="bg-pics"
  />
  <div
    className="caption-div "
  >
    <h1
      className="caption mb-4 mt-5"
    >
      Order Food Fast!
    </h1>
    <p
      className="caption-small lead"
    >
      Get your food in minutes!
    </p>
  </div>
  <div>
    <h1
      className="order"
    >
      Available Order
    </h1>
  </div>
  <Connect(withRouter(WithAlert(ProductTable))) />
  <Connect(withRouter(WithAlert(ProductCard))) />
</div>
`;
